test/infrastructure/vm-operator/vm-operator-pull-policy.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vmware-system-vmop-controller-manager
+  namespace: vmware-system-vmop
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          imagePullPolicy: Always
```

Comment:
This aligns vm operator to all the other providers; TBD if we prefer if not present or if we want to switch to if not present in our E2E test config